vaadin-menu-bar-flow-parent/vaadin-menu-bar-flow/src/main/resources/META-INF/resources/frontend/menubarConnector.js

```diff
@@ -31,6 +31,18 @@
         return;
       }
 
+      const observer = new MutationObserver((records) => {
+        const hasChangedAttributes = records.some((entry) => {
```

Comment:
**note:** I had to add this check because the context-menu has a logic where it ensures the `disabled` attribute by re-assigning it which triggers the Mutation Observer here.

https://github.com/vaadin/web-components/blob/9b9fa8a6e26d6154cb06b27c1ddd3bda6e21e1df/packages/context-menu/src/vaadin-contextmenu-items-mixin.js#L229